.circleci/config.yml

```diff
@@ -78,5 +79,5 @@ jobs:
           command: |
             . .venv/bin/activate
             pytest --cov-report= --cov=polling_stations
-            coverage run --append manage.py harvest
+            coverage run 
```

Comment:
I think you need to delete this whole line to get CI working

Reply:
Sorted :)